### ISSUE TITLE
Releases not appearing on latest dev branch

### ISSUE THREAD

**Issue by nightah** [NNTmux/newznab-tmux#139]
OS: Ubuntu LTS 16.04.2
PHP: 7.1
WS: nginx

How to reproduce issue:

- Browse to any category and no releases will appear
- Releases appear within Admin -> Releases -> View Releases and can be edited

When browsing the categories as per above the following php notices are also thrown:

```
[14-May-2017 18:52:14 Australia/Melbourne] PHP Notice:  Undefined index: pagertotalitems in /srv/http/NNTmux/resources/smarty/templates_c/eb0e7ac3c957886ef579cbb74609274fadfb4934_0.file.pager.tpl.php on line 24
[14-May-2017 18:52:14 Australia/Melbourne] PHP Notice:  Trying to get property of non-object in /srv/http/NNTmux/resources/smarty/templates_c/eb0e7ac3c957886ef579cbb74609274fadfb4934_0.file.pager.tpl.php on line 24
```

Alternatively I can go to the New Releases section:
- Click on a Movie or TV that appears within new releases

php_errors_web spits the following stack trace:
```
[14-May-2017 14:29:39 Australia/Melbourne] PHP Fatal error:  Uncaught lithium\data\model\QueryException: DESCRIBE `release_regexes`: Table 'nntmux.release_regexes' doesn't exist in /srv/http/NNTmux/libraries/lithium/data/source/Database.php:1601
Stack trace:
#0 /srv/http/NNTmux/libraries/lithium/aop/Filters.php(231): lithium\data\source\Database->lithium\data\source\{closure}(Array)
#1 /srv/http/NNTmux/libraries/lithium/data/source/Database.php(1602): lithium\aop\Filters::run(Object(lithium\data\source\database\adapter\MySql), '_error', Array, Object(Closure))
#2 /srv/http/NNTmux/libraries/lithium/data/source/database/adapter/MySql.php(406): lithium\data\source\Database->_error('DESCRIBE `relea...')
#3 /srv/http/NNTmux/libraries/lithium/aop/Filters.php(231): lithium\data\source\database\adapter\MySql->lithium\data\source\database\adapter\{closure}(Array)
#4 /srv/http/NNTmux/libraries/lithium/data/source/database/adapter/MySql.php(409): lithium\aop\Filters::run(Object(lithium\data\source\database\adapter\MySql), '_execute', Array, Obj in /srv/http/NNTmux/libraries/lithium/data/source/Database.php on line 1601
```

**Comment by DariusIII** [comment]
Wrong table name, it should be releases_regexes:
https://github.com/NNTmux/newznab-tmux/blob/0.x/resources/db/schema/mysql-ddl.sql#L832-L841

**Comment by nightah** [comment]
So looking at the actual database, the table name is correct as per your post. Having said that, the table is also empty.

Where/how is the lithium library getting the incorrect table name referenced?

**Comment by nightah** [comment]
Fixed by f85baf623b059e83ff5d3bb690d3a69af9738146
